pkg/k8sutil/k8sutil.go

```diff
@@ -54,6 +63,9 @@ func InitOperatorService() (*v1.Service, error) {
 	if err != nil {
 		return nil, err
 	}
+	if namespace == "" { // This will happen for clusterScoped operators
+		namespace, _ = GetNamespace()
```

Comment:
Check this error.

Reply:
Done. +some unit tests
Should I squash both commits?